### PR TITLE
Added LoggerRegistry

### PR DESCRIPTION
**Pull request by ewoutp**
This PR adds a registry of loggers.
It can be used to create additional loggers, get names of all created loggers and get/set log levels of these loggers.

In additional this PR adds the ability to include the logger name into the log format.

### CODE REVIEW

**Comment by ewoutp** [comment]
@ZeissS RFR

File: logger_registry.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import (
+	"fmt"
+	"sync"
+
+	"github.com/op/go-logging"
+)
+
+// LoggerRegistry provides the api for a registry of loggers.
+type LoggerRegistry interface {
+	MustCreate(name string, level ...string) Logger
```
Comment (zeisss) [inline logger_registry.go:15]: what is the purpose of these levels? The provided implementation only allows 1 parameter

File: logger_registry.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	} else if len(level) > 1 {
+		panic("Can only have 1 log level")
+	}
+	l := MustGetLogger(config)
+	lg.loggers[name] = l
+
+	return l
+}
+
+// Get returns the logger with given name or NotFoundError if no such logger
```
Comment (zeisss) [inline logger_registry.go:65]: can we put documentation on the interface too?

File: logger_registry.go (hunk 1 above)
Comment (ewoutp) [inline logger_registry.go:15]: It's an optional field. I'll add a comment

Comment (ewoutp) [inline logger_registry.go:15]: Done

File: logger_registry.go (hunk 2 above)
Comment (ewoutp) [inline logger_registry.go:65]: Yes, done

**Comment by zeisss** [comment]
LGTM

**Pull request merged**